src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -75,6 +77,13 @@ def _create_ui_components(self):
                 schema=element,
             )
 
+    def _create_integration_feature(self):
+        # sentry apps must have at least one feature
+        # defaults to 'api'
+        IntegrationFeature.objects.create(
```

Comment:
This isn't caring about duplicates, is it? Isn't that less than ideal if there were more than 1 IntegrationFeature paired with a SentryApp?

Reply:
well you can have up to four types of features associated with a sentry app, but yeah we should make it so you can't add the same type twice - that should probably be added to the model right?

Reply:
Yeah, does that mean you'd have a UNIQUE on `(sentry_app, feature)`  ?

Reply:
Yeah would need to add that

Reply:
@mattrobenolt https://github.com/getsentry/sentry/pull/13573